web/client/plugins/Swipe.jsx

```diff
@@ -0,0 +1,63 @@
+/*
+* Copyright 2020, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import { getSelectedLayer } from '../selectors/layers';
+import { layerSwipeSettingsSelector } from '../selectors/swipe';
+import swipe from '../reducers/swipe';
+import * as epics from '../epics/swipe';
+
+import { createPlugin } from '../utils/PluginsUtils';
+
+import SwipeSettings from './SwipeSettings';
+import SliderSwipeSupport from '../components/map/openlayers/swipe/SliderSwipeSupport';
+
+const Support = ({ map, layer, active }) => {
+    return <SliderSwipeSupport map={map} layer={layer} active={active} />;
+};
```

Comment:
We have to isolate somehow this implementation in a separate component. "SwipeSupport" and wrap into a generic components that loads the OL implementation on demand.
But we can do it in a separate PR. @allyoucanmap, what do you think ? Can we do in a next separated PR ?

Reply:
@offtherailz here the availability of plugin it's seem solved with disablePluginIf, (imo) if there is still work to do on this Support implementation we could improve this later on a new PR.